dev.sh

```diff
@@ -1,5 +1,46 @@
 #!/bin/bash
 
+REQUIRED_NODE_VERSION=22.9.0
```

Comment:
I think we're not strict on Node 22.9. It can be anything above 22 (for now).

Reply:
Yarn does need to be 1.22.22 at least.

Reply:
@jayantbh Sir, Should i remove yarn version check

Reply:
or please tell me minimum which version will be needed.